packages/ssz/src/util/json.ts

```diff
@@ -8,7 +8,7 @@ export function toExpectedCase(
 ): string {
   if (customCasingMap && customCasingMap[value]) return customCasingMap[value];
   switch (expectedCase) {
-    case "notransform":
+    case "declared":
```

Comment:
Remove

Reply:
this is when the field is not present in the casingMap